freqtrade/freqtradebot.py

```diff
@@ -1181,7 +1197,7 @@ def _notify_sell(self, trade: Trade, order_type: str, fill: bool = False) -> Non
             'current_rate': current_rate,
             'profit_amount': profit_trade,
             'profit_ratio': profit_ratio,
-            'sell_reason': trade.sell_reason,
+            'sell_reason': trade.sell_reason,  # TODO-lev: change to exit_reason
```

Comment:
i know you see this as helpful for future changes - but it's actually not (doesn't provide any value) and will even mean additional work for whoever will do this change.
Having this there once (usually where it's defined -> in `models.py`) will suffice.

Whoever will rename it will anyway have to search through all all files (can't rely on these comments to capture all cases 100%) - so this is actually adding work to whoever does the refactoring, as it's not just a "replace" - but also a "remove unnecessary comment" - which is a few more key-strokes for each line that needs to be changed.